src/main/java/org/example/daos/EmployeeDao.java

```diff
@@ -0,0 +1,73 @@
+package org.example.daos;
+
+import org.example.models.Employee;
+import org.example.models.EmployeeRequest;
+import java.sql.Connection;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.sql.ResultSet;
+import java.sql.PreparedStatement;
+import java.util.ArrayList;
+import java.util.List;
+
+public class EmployeeDao {
+
+    public List<Employee> getAllEmployees() throws SQLException {
```

Comment:
This function doesn't look to be used as part of this user story/requirement. Lets remove it for now

Reply:
commented out. did first by error